packages/components/src/components/link-tile-model-signature/link-tile-model-signature.examples.md

```diff
@@ -0,0 +1,188 @@
+
+<TableOfContents></TableOfContents>
+
+## Basic
+
+An `img` or `picture` tag has to be provided in the slot of the `p-link-tile-model-signature` component.
+
+To work properly the `p-link-tile-model-signature` component needs two `p-link` components as named slots,
+`slot="primary"` and `slot="secondary"` which are mandatory. Also the `heading` property is required. It is used as a
+teaser with a more detailed description of where the link leads to.
+
+<Playground :markup="basic" :config="config"></Playground>
+
+### <A11yIcon></A11yIcon> Accessibility hints
+
+In general, placing textual contents above an image can easily lead to **contrast issues**. Though we are providing a
+scalable background gradient to reduce the risk of low contrasts between foreground text and background image, there
+still can occur issues with color contrast ratios, especially if different aspect ratios for multiple viewport sizes are
+used. So, always **check readability** and play around with the `weight` property to achieve the best results.
+
+To provide more contextual HTML semantics you can use the `headingTag` property to change the heading parent from
+default `h2` to e.g. `h3`.
+
+---
+
+## Description
+
+If your heading needs further explanation or you want to provide additional information like e.g. a price tage, use the
+`description` property.
+
+<Playground :markup="description" :config="config"></Playground>
+
+---
+
+## Weight
+
+The `weight` property changes the font weight of the heading.
+
+<Playground :markup="weightMarkup" :config="config">
+  <SelectOptions v-model="weight" :values="weights" name="weight"></SelectOptions>
+</Playground>
+
+---
+
+## Link Direction
+
+The `link-diretion` property defines the direction of the main and cross axis of the links. The default is "{base:
+'row', xs: 'column'}" showing buttons vertically stacked on mobile viewports and side-by-side in a horizontal row from
+breakpoint 'xs'.
+
+<Playground :markup="linkDirectionMarkup" :config="config">
+  <SelectOptions v-model="linkDirection" :values="linkDirections" name="linkDirection"></SelectOptions>
+</Playground>
+
+---
+
+## Aspect Ratio
+
+The component takes its height from the width provided and places the image via CSS `object-fit: cover`. Therefore, you
+can change the height by using different aspect ratios.
+
+<Playground :markup="aspectRatioMarkup">
+  <SelectOptions v-model="aspectRatio" :values="aspectRatios" name="aspectRatio"></SelectOptions>
+</Playground>
+
+---
+
+## Model
+
+<Playground :markup="modelMarkup">
+  <SelectOptions v-model="model" :values="models" name="model"></SelectOptions>
+</Playground>
+
+<script lang="ts">
+import Vue from 'vue';
+import Component from 'vue-class-component';
+import { TILE_WEIGHTS, TILE_ASPECT_RATIOS } from '../../utils'; 
+import { MODEL_SIGNATURE_MODELS } from '../model-signature/model-signature-utils'; 
+import { GROUP_DIRECTIONS } from '../../styles/group-direction-styles'; 
+
+@Component
```

Comment:
How to use this component with a routing library in the supported frameworks?

Reply:
This is outscoped in a separated ticket. Was confirmed by Stephan that its not in the scope of this one.